tests/View/ViewFactoryTest.php

```diff
@@ -264,22 +264,34 @@ public function testSectionMultipleExtending()
     public function testSingleStackPush()
     {
         $factory = $this->getFactory();
-        $factory->startSection('foo');
+        $factory->startPush('foo');
         echo 'hi';
-        $factory->appendSection();
-        $this->assertEquals('hi', $factory->yieldContent('foo'));
+        $factory->stopPush();
+        $this->assertEquals('hi', $factory->yieldPushContent('foo'));
     }
```

Comment:
Since `@push` uses startPush, so this test whether the startPush, stopPush, yieldPushContent works